packages/terra-framework-docs/src/terra-dev-site/doc/tabs/example/IconOnlyTabs.jsx

```diff
@@ -1,47 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
 import classNames from 'classnames/bind';
 import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
 import IconSearch from 'terra-icon/lib/icon/IconSearch';
-import IconBookmark from 'terra-icon/lib/icon/IconBookmark';
-import IconCalendar from 'terra-icon/lib/icon/IconCalendar';
 import Tabs from 'terra-tabs';
 import TabContent from './TabContentTemplate';
 import styles from './common/TabExample.module.scss';
 
 const cx = classNames.bind(styles);
 
+let i = -1;
 const IconOnlyTabs = () => {
-  const searchTab = (
-    <Tabs.Pane label="Search" icon={<IconSearch />} isIconOnly key="Search">
-      <TabContent label="Search" />
-    </Tabs.Pane>
-  );
+  const tabsExample = [];
+  const [tabs, setTabs] = useState(tabsExample);
+  const [activeKey, setActiveKey] = useState();
 
-  const briefcaseTab = (
-    <Tabs.Pane label="Briefcase" icon={<IconBriefcase />} isIconOnly key="Briefcase">
-      <TabContent label="Briefcase" />
-    </Tabs.Pane>
+  tabsExample.push(
+    {
+      label: 'Search',
+      icon: <IconSearch />,
+      isIconOnly: true,
+      key: 'Search',
+      content: 'label',
+    },
   );
-
-  const bookmarkTab = (
-    <Tabs.Pane label="Bookmark" icon={<IconBookmark />} isIconOnly key="Bookmark">
-      <TabContent label="Bookmark" />
-    </Tabs.Pane>
+  tabsExample.push(
+    {
+      label: 'Briefcase',
+      icon: <IconBriefcase />,
+      isIconOnly: true,
+      key: 'Briefcase',
+      content: 'label',
+    },
   );
 
-  const calendarTab = (
-    <Tabs.Pane label="Calendar" icon={<IconCalendar />} isIconOnly key="Calendar">
-      <TabContent label="Calendar" />
-    </Tabs.Pane>
+  tabsExample.push(
+    {
+      label: 'Tab with Text',
+      isIconOnly: false,
+      key: 'TabWithText',
+      content: 'Tab with Text',
+    },
   );
+  const handleTabClose = (tabdata, itemKey) => {
+    const tabsArray = [...tabs];
+    const indexToRemove = tabsArray.findIndex(tab => tab.key === itemKey);
+    if (indexToRemove !== -1) {
+      tabsArray.splice(indexToRemove, 1);
+      setTabs(tabsArray);
+    }
+  };
+
+  const addMoreTabPanes = () => {
+    i += 1;
+    const tabsArray = tabs;
+    tabsArray.push(
+      {
+        label: `Tab_${i}`,
+        isIconOnly: false,
+        key: `Tab_${i}`,
+        content: `Tab_${i}`,
+        isActive: true,
+      },
+    );
+    setActiveKey(`Tab_${i}`);
+    setTabs(tabsArray);
+  };
 
   return (
     <div className={cx('content-wrapper')}>
-      <Tabs>
-        {searchTab}
-        {briefcaseTab}
-        {bookmarkTab}
-        {calendarTab}
+      <Tabs activeKey={activeKey} isDraggable isClosable onSelectAddButton={addMoreTabPanes} ariaLabelAddTab="Add Tab" onTabClose={handleTabClose}>
+        { tabs.map((tab) => (
+          <Tabs.Pane label={tab.label} isIconOnly={tab.isIconOnly} icon={tab.icon} key={tab.key} id={tab.key} isActive={TabContent.isActive}>
```

Comment:
```suggestion
          <Tabs.Pane label={tab.label} isIconOnly={tab.isIconOnly} icon={tab.icon} key={tab.key} id={tab.key} isActive={tab.isActive}>
```
This should be `tab` instead of TabContent

Reply:
Done https://github.com/cerner/terra-framework/commit/09f89f4169db28da2af3774ea6bd197df5344813